src/components/screen/screen-form.jsx

```diff
@@ -108,6 +163,36 @@ function ScreenForm({
             onChange={handleInput}
           />
         </ContentBody>
+        {Object.prototype.hasOwnProperty.call(screen, "@id") && (
+          <ContentBody>
+            <h2 className="h4 mb-3">{t("screen.bind-header")}</h2>
+            {screen?.screenUser && (
+              <>
+                <div className="text-success mb-3">
```

Comment:
Remove text-success, it's nok accessible

Reply:
fixed